crowbar_framework/app/views/installer/index.html.haml

```diff
@@ -0,0 +1,25 @@
+.col-lg-6.col-md-6.col-sm-6.col-xs-12
+  .panel.panel-default
+    .panel-heading
+      %h2
+        = t(".installation_header").html_safe
+        .btn-group.pull-right
+          = button_to "Install", install_path, class: "btn btn-success", method: :post
+    %ul.list-group.action
+      - @steps.each do |step|
+        %li.list-group-item{ class: step }
+          = icon_tag(:hourglass_o)
+          = t(".#{step}")
+.col-lg-5.lg-offset-1.col-md-5.col-md-offset-1.col-sm-6.col-xs-12
+  %h1 Documentation
+  %ul.list-unstyled
+    %li
+      %a{:href => "#"}
+        Release Notes
+        %small
+          %i.fa.fa-external-link
+    %li
+      %a{:href => "#"}
```

Comment:
Hash attribute should start with one space after the opening brace<br>Hash attribute should end with one space before the closing brace